app/styles/partials/utils.scss

```diff
@@ -236,4 +236,17 @@
 
 .mr-8 {
   margin-right: 2rem !important;
+  margin-bottom: -120px;
```

Comment:
mr here means margin-right..do not change anything in existing classes.

Reply:
okay will revert this